presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -465,7 +467,7 @@ public void createView(ConnectorSession session, ConnectorTableMetadata viewMeta
                 .setPartitionColumns(ImmutableList.of())
                 .setParameters(properties)
                 .setViewOriginalText(Optional.of(encodeViewData(viewData)))
-                .setViewExpandedText(Optional.of("/* Presto View */"));
+                .setViewExpandedText(Optional.of(PRESTO_VIEW_EXPANDED_TEXT_MARKER));
```

Comment:
Can you please merge this commit with the original change? Each commit shall handle a problem, but it cannot be "address comments".

Reply:
Sure.

Reply:
I had pushed a separate commit so that it is easier to review the changes incrementally @yingsu00 
After review is done, I can squash the commits.